lib/splash.js

```diff
@@ -39,7 +41,13 @@ function logKey(key) {
 }
 
 function logValue(key, value) {
-    return "| " + indent() + " " + key + ": " + value + "\n";
+    var hideValue = _.reduce(KEYS_TO_HIDE, function(memo, keyToHide) {
+        return memo || key.indexOf(keyToHide) >= 0;
```

Comment:
In the case of an array, the key corresponds to the index in the array. Here is what splash displayed before your modification:
```
| -- locales
| ---- 0: en
| ---- 1: fr
```